app/Models/User.php

```diff
@@ -42,7 +43,10 @@ class User extends Authenticatable
 
     public function toFlare(): array
     {
-        return $this->only(['level', 'auth_type', 'enabled']);
+        $user = $this->only(['level', 'auth_type', 'enabled']);
+        $user['id'] = ErrorReportingProvider::getInstanceId();
```

Comment:
This is this wrong place to inject it I've realized. Doesn't work with poller since we got no user instance then. Should be Middleware